include/LevenbergAlgorithm2.h

```diff
@@ -96,6 +96,7 @@ class LevenbergAlgorithm : public Algorithm {
       double m_StepToleranceMaximum = 0;
 
       int m_LambdasMinimum = 1;
+      int m_DerivativeType = 2;
```

Comment:
Maybe add in comment which Type=2 actually is (forward/backward/central)?

Reply:
Good idea. I forgot which that was by the time I went back and looked. I fixed this in the changes I just made.